components/eamxx/scripts/utils.py

```diff
@@ -376,9 +376,20 @@ def package_version_ok(pkg, min_version=None):
     Checks that the loaded package's version is >= that the minimum required one.
     If no minimum version is passed, then return True
     """
-    from pkg_resources import parse_version
+    if min_version is not None:
+        try:
+            from pkg_resources import parse_version
+
+            return parse_version(pkg.__version__) >= parse_version(min_version)
+        except ImportError:
+            # Newer versions of python cannot use pkg_resources
+            ensure_packaging()
+            from packaging.version import parse
 
-    return True if min_version is None else parse_version(pkg.__version__) >= parse_version(min_version)
+            return parse(pkg.__version__) >= parse(min_version)
```

Comment:
We may just do
```py
[int(i) for i in my_mod.__version__.split('.')] >= [int(i) for i in min_version.split('.')]
```
and avoid importing a new module...

Reply:
Yeah, that crossed my mind too. I think there is maybe more nuance than meets the eye. For example, if the lists are not the same length, your code won't work. There's probably a fair bit of error handling too. We already have a few external deps, so I think it's not a big deal to have another small one.